.azure-pipelines/templates/installer-tests.yml

```diff
@@ -0,0 +1,31 @@
+jobs:
+  - job: installer
+    pool:
+      vmImage: vs2017-win2016
+    steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: 3.7
+          architecture: x86
+          addToPath: true
+      - script: python windows-installer/construct.py
+        displayName: Build Certbot installer
+      - task: CopyFiles@2
+        inputs:
+          sourceFolder: $(System.DefaultWorkingDirectory)/windows-installer/build/nsis
+          contents: '*.exe'
+          targetFolder: $(Build.ArtifactStagingDirectory)
+      - task: PublishPipelineArtifact@1
+        inputs:
+          path: $(Build.ArtifactStagingDirectory)
+          artifact: WindowsInstaller
+      - script: python windows-installer/install.py
+        displayName: Install Certbot
+      - script: |
+          python -m venv venv
+          venv\Scripts\python tools\pip_install.py -e certbot-ci
+        displayName: Prepare Certbot-CI
+      - script: |
+          set PATH=%ProgramFiles(x86)%\Certbot\bin;%PATH%
```

Comment:
Do you know why we need to do this? The installer should be putting `certbot` in the default `PATH`. It'd be nice if we were able to rely on this behavior and help test it here.

Reply:
It fails without it. Yes normally the installer adds the relevant entries, and it works on all real systems I tested, except in CI. For your information, the tests I made on Travis end up with the same issue.

There is something weird with the env set by CIs, so far I did not find the underlying reason.

Reply:
I couldn't find a better way either. I suspect it's something like the pipeline is being executed in one continuous environment that's not reloaded to pick up changes like this.

I searched around for a way to explicitly reset things like environment variables to their default value on Windows and couldn't find anything.

Reply:
I tried dedicated Powershell and Batch dedicated utility provided by choco for that, https://chocolatey.org/docs/helpers-update-session-environment, and also another explicit command to feed PATH from the Registry that I do not find again at the moment.

But all without success.

Reply:
Weird...

Thanks for trying all of that.